src/app/static/src/app/components/plots/MapLegend.vue

```diff
@@ -103,7 +103,7 @@
                         val = roundToContext(val, [min, max]);
 
                         let valAsProportion = (max != min) ? (val - min) / (max - min) : 0;
-                        if (this.metadata.invert_scale) {
+                        if (this.metadata?.invert_scale) {
```

Comment:
For some reason even though this is [nested in a falsy check](https://github.com/mrc-ide/hint/pull/614/files#diff-44d66b7e90e2c4ea3a07ea59621b8af2c92f4187dab4d157187021b446a0d8f9L91) the compiler doesn't pick up that `metadata` must be defined here.

Reply:
Can you reference the `this` object within this method? I'm not sure if this counts as an anonymous function or not. You could try putting `this.metadata` into its own variable and referencing that instead.

Reply:
Great shout, I will do this. Maybe this has never worked!?

Reply:
Oh actually, using the arrow function syntax means that `this` is preserved within `map` functions. So this does seem like a compiler mistake. But I'll still pull out the variable.